backend/app/controllers/app.py

```diff
@@ -251,7 +252,19 @@ def generate_for_all_models():
         models = get_files_in_folder(curr_user)
         for i in range(len(models)):
             models[i] = os.path.join(UPLOAD_FOLDER, curr_user, models[i])
-        print(models)
+        print("Multiple Models Paths:", models)
+
+        evaluator = EvaluateModelsUseCase(models)
+        output = evaluator.execute()
+
+        print("Multiple Models Output", output)
+
+        output_filtered = []
```

Comment:
is output_filtered the elements of the dictionary?

Reply:
The return statement returns a dictionary with keys but Frotnend expects a list of simple dictionaries. So I had to filter out the key values for the correct JSON return